qt/applications/workbench/workbench/plotting/toolbar.py

```diff
@@ -30,27 +30,27 @@ class WorkbenchNavigationToolbar(NavigationToolbar2QT):
     sig_waterfall_conversion = QtCore.Signal(bool)
 
     toolitems = (
-        ('Home', 'Center display on contents', 'mdi.home', 'on_home_clicked', None),
+        ('Home', 'Reset axes limits', 'mdi.home', 'on_home_clicked', None),
         ('Back', 'Back to previous view', 'mdi.arrow-left', 'back', None),
         ('Forward', 'Forward to next view', 'mdi.arrow-right', 'forward', None),
         (None, None, None, None, None),
-        ('Pan', 'Pan axes with left mouse, zoom with right', 'mdi.arrow-all', 'pan', False),
-        ('Zoom', 'Zoom to rectangle', 'mdi.magnify', 'zoom', False),
+        ('Pan', 'Pan: L-clk \nStretch: R-clk', 'mdi.arrow-all', 'pan', False),
+        ('Zoom', 'Zoom \n In: L-clk+drag \n Out: R-clk+drag', 'mdi.magnify', 'zoom', False),
```

Comment:
I don't like L-clk and R-clk, I think just
```suggestion
        ('Pan', 'Pan: L-click \nStretch: R-click', 'mdi.arrow-all', 'pan', False),
        ('Zoom', 'Zoom \n In: L-click+drag \n Out: R-click+drag', 'mdi.magnify', 'zoom', False),
```
works fine.

Reply:
happy to change to click!